src/views/v2/commerce/checkout/order-modifiers/coupons.php

```diff
@@ -3,32 +3,77 @@
-	</button>
-	<p id="coupon_error" class="tribe-tickets__commerce-checkout-cart-coupons__error" style="display: none; color: red;" aria-live="polite" role="alert">
-		<?php esc_html_e( 'Invalid Coupon Code', 'event-tickets' ); ?>
+<div class="tribe-tickets__form tribe-tickets__commerce-checkout-coupons-wrapper tribe-common-b2">
+	<div class="tec-tickets__commerce-checkout-cart-coupons" <?php echo $hide_input ? 'style="display: none;"' : ''; ?>>
+		<input
+			type="text"
+			class="<?php tribe_classes( $coupon_input_classes ); ?>"
+			id="coupon_input"
+			name="coupons"
+			aria-describedby="coupon_error"
+			aria-label="<?php esc_attr_e( 'Enter coupon code', 'event-tickets' ); ?>"
+			placeholder="<?php esc_attr_e( 'Enter coupon code', 'event-tickets' ); ?>"
+			value="<?php echo esc_attr( $coupon['slug'] ?? '' ); ?>"
+		/>
+		<button
+			id="coupon_apply"
+			class="<?php tribe_classes( $apply_button_classes ); ?>"
+		>
+			<?php echo esc_html_x( 'Apply', 'button to apply a coupon code', 'event-tickets' ); ?>
+		</button>
+	</div>
+	<p id="coupon_error" class="tec-tickets__commerce-checkout-cart-coupons__error" style="display: none; color: red;" aria-live="polite" role="alert">
+		<?php esc_html_e( 'Invalid coupon code', 'event-tickets' ); ?>
 	</p>
-</div>
-<div class="tribe-tickets__commerce-checkout-cart-coupons__applied" style="display: none;">
-	<span class="tribe-tickets__commerce-checkout-cart-coupons__applied-text">
-		<?php esc_html_e( 'Coupon:', 'event-tickets' ); ?> <span class="tribe-tickets__commerce-checkout-cart-coupons__applied-value"></span> - <?php esc_html_e( 'Discount:', 'event-tickets' ); ?> <span class="tribe-tickets__commerce-checkout-cart-coupons__applied-discount"></span>
-	</span>
-	<button class="tribe-tickets__commerce-checkout-cart-coupons__remove-button" type="button">
-		<?php esc_html_x( 'X', 'text for button to remove an applied coupon', 'event-tickets' ); // @todo use an ICON instead. @see https://github.com/the-events-calendar/event-tickets/pull/3430#discussion_r1872545653 ?>
-	</button>
+	<div class="tec-tickets__commerce-checkout-cart-coupons__applied" <?php echo $hide_applied ? 'style="display: none;"' : ''; ?>>
+		<span class="tec-tickets__commerce-checkout-cart-coupons__applied-text">
+			<?php esc_html_e( 'Coupon:', 'event-tickets' ); ?>
+			<span class="tec-tickets__commerce-checkout-cart-coupons__applied-value">
+				<?php echo esc_html( $coupon['display_name'] ?? '' ); ?>
+			</span>
+			- <?php esc_html_e( 'Discount:', 'event-tickets' ); ?>
+			<span class="tec-tickets__commerce-checkout-cart-coupons__applied-discount">
+				<?php echo esc_html( $discount ); ?>
+			</span>
+		</span>
+		<button class="tec-tickets__commerce-checkout-cart-coupons__remove-button" type="button">
+			<?php echo esc_html_x( 'X', 'text for button to remove an applied coupon', 'event-tickets' ); // @todo use an ICON instead. @see https://github.com/the-events-calendar/event-tickets/pull/3430#discussion_r1872545653 ?>
```

Comment:
Get this `@TODO` out of here and do it.

Reply:
https://github.com/the-events-calendar/tribe-common/blob/8de5260664aef59e4d826bfdfec0a283c93bea31/src/resources/images/icons/close.svg

Reply:
You know, I spotted this at one point, but then I forgot about it. It's so far off the right side of the line that I missed it when scrolling through the file later 🤦🏻‍♂️ 

Thanks for the link, I'll update it with the SVG.